pop-api/examples/dao/src/tests.rs

```diff
@@ -0,0 +1,549 @@
+fn vote_fails_if_voting_period_ended(mut session: Session) {
+	let _ = env_logger::try_init();
+	// Deploy a new contract.
+	let contract = deploy_with_default(&mut session).unwrap();
+	// Prepare voters accounts
+	let _ = prepare_dao(&mut session, contract.clone());
+
+	// Alice create a proposal
+	let description = "Funds for creation of a Dao contract".to_string().as_bytes().to_vec();
+	let amount = AMOUNT * 3;
+	session.set_actor(ALICE);
+	let mut call_f = Vec::new();
+	let call0 = RuntimeCall::Fungibles(FungiblesCall::TransferFrom {
+		token: TOKEN,
+		from: account_id_from_slice(&contract),
+		to: account_id_from_slice(&BOB),
+		value: amount,
+	})
+	.encode(&mut call_f);
+	assert_ok!(create_proposal(&mut session, BOB, amount, description, call_f));
+
+	// Moving to blocks beyond voting period
+	session.sandbox().build_blocks(VOTING_PERIOD + 1);
+
+	session.set_actor(CHARLIE);
+	// Charlie tries to vote
+	assert_eq!(vote(&mut session, 1, true), Err(Error::VotingPeriodEnded));
+}
+
+#[drink::test(sandbox = Pop)]
+fn vote_fails_if_not_a_member(mut session: Session) {
+	let _ = env_logger::try_init();
+	// Deploy a new contract.
+	let contract = deploy_with_default(&mut session).unwrap();
+	// Prepare voters accounts
+	let _ = prepare_dao(&mut session, contract.clone());
+
+	// Alice create a proposal
+	let description = "Funds for creation of a Dao contract".to_string().as_bytes().to_vec();
+	let amount = AMOUNT * 3;
+	session.set_actor(ALICE);
+	let mut call_f = Vec::new();
+	let call0 = RuntimeCall::Fungibles(FungiblesCall::TransferFrom {
+		token: TOKEN,
+		from: account_id_from_slice(&contract),
+		to: account_id_from_slice(&BOB),
+		value: amount,
+	})
+	.encode(&mut call_f);
+	assert_ok!(create_proposal(&mut session, BOB, amount, description, call_f));
+
+	session.set_actor(NON_MEMBER);
+	assert_eq!(vote(&mut session, 1, true), Err(Error::MemberNotFound));
+}
+
+#[drink::test(sandbox = Pop)]
+fn proposal_enactment_works(mut session: Session) {
+	let _ = env_logger::try_init();
+	// Deploy a new contract.
+	let contract = deploy_with_default(&mut session).unwrap();
+	// Mint tokens.
+	assert_ok!(session.sandbox().mint_into(&TOKEN, &contract.clone(), AMOUNT));
+	assert_ok!(session.sandbox().approve(&TOKEN, &contract.clone(), &contract.clone(), AMOUNT));
+	// Prepare voters accounts
+	let _ = prepare_dao(&mut session, contract.clone());
+
+	// Alice create a proposal
+	let description = "Funds for creation of a Dao contract".to_string().as_bytes().to_vec();
+	let amount = MIN_BALANCE;
+	session.set_actor(ALICE);
+	let mut call_f = Vec::new();
+	let call0 = RuntimeCall::Fungibles(FungiblesCall::TransferFrom {
+		token: TOKEN,
+		from: account_id_from_slice(&contract),
+		to: account_id_from_slice(&BOB),
+		value: amount,
+	})
+	.encode(&mut call_f);
+	assert_ok!(create_proposal(&mut session, BOB, amount, description, call_f));
+
+	let now = session.sandbox().block_number();
+	session.set_actor(CHARLIE);
+	// Charlie vote
```

Comment:
```suggestion
	// CHARLIE votes
```

To keep it consistent with the below comments.